src/app/app.module.ts

```diff
@@ -1,18 +1,32 @@
-import { NgModule } from '@angular/core';
+import { InjectionToken, NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
-
-import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { BaseClassComponent } from './dependency-injection/with-inheritance/base-class/base-class.component';
+import { MyClassComponent } from './dependency-injection/with-inheritance/my-class/my-class.component';
+import { BaseClassForInjectorComponent } from './dependency-injection/using-injector/base-class-for-injector/base-class-for-injector.component';
+import { MyClassForInjectorComponent } from './dependency-injection/using-injector/my-class-for-injector/my-class-for-injector.component';
+import { MyClassForInjectMethodComponent } from './dependency-injection/using-inject-method/my-class-for-inject-method/my-class-for-inject-method.component';
+import { BaseClassForInjectMethodComponent } from './dependency-injection/using-inject-method/base-class-for-inject-method/base-class-for-inject-method.component';
+import { LoggerService } from '../shared/logger.service';
+
+export const LOGGER_TOKEN = new InjectionToken<LoggerService>('LoggerService');
+export const LOGGER_CONFIG_TOKEN = new InjectionToken<string>('loggerServiceConfig');
```

Comment:
for the future: it's better to place it to /constants folder for not having circular imports from the module file

Reply:
but here you can leave it as it is